eservice/pdo/sservice/block_store_manager.py

```diff
@@ -137,8 +137,17 @@ def get_block(self, block_id, encoding='b64') :
 
         return block_data
 
+    # return block_data_list
+    def __block_iterator__(self, block_ids, encoding) :
+        for block_id in block_ids :
+            yield self.get_block(block_id, encoding)
+
     def get_blocks(self, block_ids, encoding='b64') :
```

Comment:
where is this function called? I see that WSGI middleware function GetBlocksApp makes multiple calls of get_block instead.

Reply:
i think it is used in the final version of the file.